docs/api/checkouts.rst

```diff
@@ -238,185 +563,307 @@ Response will contain an updated order information.
+                        "price":8787
                     },
-                    "order": {
-                        "href": "/app_dev.php/api/v1/orders/52"
+                    {
+                        "id":2,
+                        "code":"dhl_express",
+                        "name":"DHL Express",
+                        "description":"Voluptatem ipsum dolor vitae corrupti eum repellat.",
+                        "price":3549
                     },
-                    "self": {
-                        "href": "/app_dev.php/api/v1/shipments/51"
+                    {
+                        "id":3,
+                        "code":"fedex",
+                        "name":"FedEx",
+                        "description":"Qui nostrum minus accusantium molestiae voluptatem eaque.",
+                        "price":3775
                     }
-                },
-                "created_at": "2014-12-15T14:30:40+0000",
-                "id": 51,
-                "method": {
-                    "_links": {
-                        "self": {
-                            "href": "/app_dev.php/api/v1/shipping-methods/4"
-                        },
-                        "zone": {
-                            "href": "/app_dev.php/api/v1/zones/4"
-                        }
-                    },
-                    "calculator": "flexible_rate",
-                    "category_requirement": 1,
-                    "configuration": {
-                        "additional_item_cost": 500,
-                        "additional_item_limit": 10,
-                        "first_item_cost": 4000
-                    },
-                    "created_at": "2014-12-03T09:54:28+0000",
-                    "enabled": true,
-                    "id": 4,
-                    "name": "FedEx World Shipping",
-                    "updated_at": "2014-12-03T09:54:28+0000"
-                },
-                "state": "checkout",
-                "updated_at": "2014-12-15T14:30:41+0000"
+                ]
             }
-        ],
-        "shipping_address": {
-        },
-        "state": "cart",
-        "total": 1504750,
-        "updated_at": "2014-12-15T14:30:41+0000",
-        "user": {
-        }
+        ]
     }
 
+Response contains the proposed shipments and for each, it also has a list of shipping methods available with calculated price.
 
-Payment step
-------------
+.. warning::
 
-When we are done with shipping choices and we know the final price of an order, we can select a payment method.
+    Because of custom calculation logic for this endpoint the regular rules of overriding does not apply for this endpoint. In order to achieve different message one has to provide custom controller and build the message by his own. Exemplary implementation can be found `here`__
+
+Next step is updating the order with the types of shipping method that one has selected. A PUT request has to be send for each available shipment,
```

Comment:
shipping method**s** that have been selected. A ``PUT`` request has to be sent for each shipment.